selendroid-standalone/src/test/java/io/selendroid/standalone/android/impl/AbstractDeviceTest.java

```diff
@@ -49,10 +49,16 @@ public void testGetCrashLogContents() {
     when(device.getCrashLog()).thenCallRealMethod();
     when(
         device.executeCommandQuietly(argThat(matchesCmdLine("adb shell ls /storage/"))))  // The trailing '/' is key
-        .thenReturn("some_file\nappcrash.log\nanother_file");
+        .thenReturn(String.format("some_file%nappcrash.log%nanother_file"));
+    when(
+        device.executeCommandQuietly(argThat(matchesCmdLine("adb.exe shell ls C:\\storage/"))))  // The trailing '/' is key
```

Comment:
are you saying this should be `adb.exe shell ls /storage/` instead?

could `matchesCmdLine()` take a regular expression and avoid the duplication of code?  for example:

```Java
    when(
         device.executeCommandQuietly(argThat(matchesCmdLine("adb(.exe)? shell ls /storage/"))))  // The trailing '/' is key
        .thenReturn(String.format("some_file%nappcrash.log%nanother_file"));
```